zhaquirks/terncy/pp01.py

```diff
@@ -0,0 +1,103 @@
+"""Device handler for Terncy awareness switch."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Identify,
+    Ota,
+    PollControl,
+    PowerConfiguration,
+)
+from zigpy.zcl.clusters.measurement import (
+    TemperatureMeasurement,
+    IlluminanceMeasurement,
+    OccupancySensing,
+)
+
+from zhaquirks import DoublingPowerConfigurationCluster
+
+from . import (
+    IlluminanceMeasurementCluster,
+    TemperatureMeasurementCluster,
+    TerncyRawCluster,
+    OccupancyCluster,
+    MotionClusterLeft,
+    MotionClusterRight,
+    BUTTON_TRIGGERS,
+)
+from .. import Bus
+from ..const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+TERNCY_AWARENESS_DEVICE_TYPE = 0x01F0
+
+
+class TerncyAwarenessSwitch(CustomDevice):
+    """Terncy awareness switch."""
+
+    def __init__(self, *args, **kwargs):
+        """Init."""
+        self.motion_left_bus = Bus()
+        self.motion_right_bus = Bus()
+        self.occupancy_bus = Bus()
+        super().__init__(*args, **kwargs)
+
+    signature = {
+        #  <SimpleDescriptor endpoint=1 profile=260 device_type=496
+        #  device_version=0
+        #  input_clusters=[0, 1, 3, 32, 1024, 1026, 1030, 64716]
+        #  output_clusters=[25]>
+        MODELS_INFO: [("Xiaoyan", "TERNCY-PP01"), (None, "TERNCY-PP01")],
```

Comment:
@ilarrain any reason why you are not defining manufacturer for the second tuple in `MODELS_INFO`???

Reply:
Yes, the reason is that I have 3 Terncy devices, 2 Awareness Switch and 1 knob, the first ones correctly report the manufacturer as Xiaoyan, and the latter one (the knob) just reports None polling the attribute on the Basic cluster (and zha reports as unk_manufacturer). I assumed "TERNCY-XXX" is sufficiently unique.